test/soak_test/start-server.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+HAZELCAST_TEST_VERSION="4.0.2"
+HAZELCAST_VERSION="4.0.2"
+
+CLASSPATH="../../hazelcast-${HAZELCAST_VERSION}.jar:../../hazelcast-${HAZELCAST_TEST_VERSION}-tests.jar"
+CMD_CONFIGS="-Dhazelcast.multicast.group=224.206.1.1 -Djava.net.preferIPv4Stack=true"
+java ${CMD_CONFIGS} -cp ${CLASSPATH} \
+    # uncomment for Java 9+ compatibility
+    # --add-modules java.se --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.management/sun.management=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED \
+    com.hazelcast.core.server.HazelcastMemberStarter \
+    > hazelcast-${HAZELCAST_VERSION}-out.log 2>hazelcast-${HAZELCAST_VERSION}-err.log &
```

Comment:
Noticed this after the approval, but it seems that we are using the same file name for the logs. (We will be running 2 members per machine). That would probably result in a single log file instead of 2 per machine

Reply:
Good idea. Let me check what can be done here.

Reply:
I've added pid to log file names. See 5da7e23ab5ea8e83ebf669c610b3b4b1b8f5f094